PIL/Image.py

```diff
@@ -2223,6 +2317,7 @@ def _check_size(size):
 
 
 def new(mode, size, color=0):
+    # type: (Mode, Size, Optional[Union[int, float, Tuple, Text]]) -> Image
```

Comment:
This union might make sense aliased/named as a Colorspec. (and the Tuple is going to be 1..4 of ints at this point)